esm/inverse_folding/util.py

```diff
@@ -42,16 +42,21 @@ def load_structure(fpath, chain=None):
         structure = pdb.get_structure(pdbf, model=1)
     bbmask = filter_backbone(structure)
     structure = structure[bbmask]
-    chains = get_chains(structure)
-    print(f'Found {len(chains)} chains:', chains, '\n')
-    if len(chains) == 0:
+    all_chains = get_chains(structure)
+    if len(all_chains) == 0:
         raise ValueError('No chains found in the input file.')
     if chain is None:
-        chain = chains[0]
-    if chain not in chains:
-        raise ValueError(f'Chain {chain} not found in input file')
-    structure = structure[structure.chain_id == chain]
-    print(f'Loaded chain {chain}\n')
+        chain_ids = all_chains
+    elif isinstance(chain, list):
+        chain_ids = chain
+    else:
+        chain_ids = [chain] 
+    for chain in chain_ids:
+        if chain not in all_chains:
+            raise ValueError(f'Chain {chain} not found in input file')
+    chain_filter = [a.chain_id in chain_ids for a in structure]
+    structure = structure[chain_filter]
+    print(f'Loaded chains {chain_ids}\n')
```

Comment:
better to avoid printing inside util functions; either move the print to top level (notebook or example script), or put `verbose=False` flag to gate this

Reply:
good point, removed :)